src/views/preview/comment/top-level-comment.jsx

```diff
@@ -96,6 +96,7 @@ class TopLevelComment extends React.Component {
             <FlexRow className="comment-container">
                 <Comment
                     highlighted={highlightedCommentId === id}
+                    key={`top-level-comment-${id}`}
```

Comment:
wasn't sure if there's a good key naming convention to use... these are sorta inconsistent, but maybe that doesn't matter